### PR TITLE
Prototype HarnessU

### PR DESCRIPTION
**Pull request by ravilach**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [x] *Optional* Screenshot. 

image

### CODE REVIEW

**Comment by ravilach** [comment]
Missed a period, re-committed 
image

File: university/certifications/static/instruqt-screen.png
Comment (jenniferopal) [inline university/certifications/static/instruqt-screen.png:1]: The file is renamed as `instruqt-*`. I'm unsure if this is a typo but if it isn't, do ignore this.

File: university/certifications/static/instruqt-time-limit.png
Comment (jenniferopal) [inline university/certifications/static/instruqt-time-limit.png:1]: This one too. The file is renamed as `instruqt-*`. I'm unsure if this is a typo but if it isn't, do ignore this.

File: university/certifications/static/instruqt-widget-scorpion.png
Comment (jenniferopal) [inline university/certifications/static/instruqt-widget-scorpion.png:1]: Finally, this one. The file is renamed as `instruqt-*`. I'm unsure if this is a typo (instruct) but if it isn't, do ignore this.

**Review by jenniferopal — COMMENTED** [review]
Finished my review for FF. Just a few comments on some typos but otherwise, it looks good to me. :)

**Comment by ravilach** [comment]
No need to review as this is being closed.

**Pull request closed without merging**